scalding-serialization/src/main/scala/com/twitter/scalding/serialization/macros/impl/ordered_serialization/providers/TraversablesOrderedBuf.scala

```diff
@@ -136,8 +136,8 @@ object TraversablesOrderedBuf {
         val b = freshT("b")
         q"""
         val $innerCompareFn = { (a: _root_.java.io.InputStream, b: _root_.java.io.InputStream) =>
-          val $a = a
-          val $b = b
+          val $a: _root_.java.io.InputStream = a
```

Comment:
I wonder if we even need this binding. It looks unused doesn’t it? Same with `$b`